src/App.css

```
@@ -139,7 +139,11 @@ form {
   list-style: none;
   padding-block: 15px;
 }
-
+/*type your text*/
```

Can you make this
```suggestion
/*contact form message input text styling*/
```

Or something else a bit more informative?